packages/invoices/src/OutboundShipment/DetailView/Footer.tsx

```diff
@@ -116,8 +116,8 @@ export const Footer: FC = () => {
                   variant="contained"
                   color="secondary"
                   onClick={async () => {
-                    success('Saved invoice! 🥳 ')();
                     await update({ status: getNextOutboundStatus(status) });
+                    success('Saved invoice! 🥳 ')();
```

Comment:
showing after it's saved. Not sure what to *actually* display?

Reply:
Haha, yes. I suppose my question was more in regard to if there should even be a notification. I originally added it just for a visual indicator while developing. Though, if there is a confirmation modal before status changes (requested by craig) .. then maybe there is a better solution (i.e. should the confirmation modal just stay open and show a loading indicator..?)

Reply:
oh, right. if there's a confirmation modal then that could stay open and spin, like you say. then close if the save is ok, or show an error if not? That ties in with the other issue of showing a blocking modal to prevent changes while updating status